tests/utilities/test_callables.py

```diff
@@ -139,33 +139,63 @@ def f(
             pass
 
         schema = callables.parameter_schema(f)
-        assert schema.dict() == {
-            "title": "Parameters",
-            "type": "object",
-            "properties": {
-                "x": {
-                    "title": "x",
-                    "type": "string",
-                    "format": "date-time",
-                    "position": 0,
-                },
-                "y": {
-                    "title": "y",
-                    "default": "2025-01-01T00:00:00+00:00",
-                    "type": "string",
-                    "format": "date-time",
-                    "position": 1,
+        if HAS_PYDANTIC_V2:
```

Comment:
I opted to check this inside the test directly so it's easy to see the differences between the v1 and v2 versions of the schema

Reply:
I like what you did with the schemas here!